clang/lib/Basic/Targets/X86.h

```diff
@@ -979,6 +979,9 @@ class LLVM_LIBRARY_VISIBILITY CygwinX86_64TargetInfo : public X86_64TargetInfo {
     if (Opts.CPlusPlus)
       Builder.defineMacro("_GNU_SOURCE");
   }
+  BuiltinVaListKind getBuiltinVaListKind() const override {
```

Comment:
```suggestion

  BuiltinVaListKind getBuiltinVaListKind() const override {
```

Odd that clang-format doesn't trigger on that